build/escargot.cmake

```diff
@@ -89,6 +89,32 @@ ADD_SUBDIRECTORY (third_party/GCutil)
 
 SET (ESCARGOT_LIBRARIES ${ESCARGOT_LIBRARIES} gc-lib)
 
+# LIBBF
+ADD_LIBRARY (libbf STATIC
+        ${ESCARGOT_THIRD_PARTY_ROOT}/libbf/libbf.c
+        ${ESCARGOT_THIRD_PARTY_ROOT}/libbf/cutils.c
+        )
+TARGET_INCLUDE_DIRECTORIES (libbf PUBLIC ${ESCARGOT_THIRD_PARTY_ROOT}/libbf)
+SET (LIBBF_CFLAGS
+	${ESCARGOT_GCUTIL_CFLAGS} # we can share arch flags with gcutil
+	${CFLAGS_FROM_ENV}
+	-w
+	-g3
+    -fdata-sections
+    -ffunction-sections
+    -fno-omit-frame-pointer
+    -fvisibility=hidden)
```

Comment:
Just a minor correction.
Would you please keep the indentation at here?

Reply:
One more thing,
It seems that only a few c files are used by Escargot. So how about excluding any other unnecessary files like `sha1sum` files (maybe checksum files?)

Reply:
Sounds great. I am going to remove unused files